tests/suites/test_suite_ssl.data

```diff
@@ -201,155 +201,160 @@ move_handshake_to_state:MBEDTLS_SSL_IS_SERVER:MBEDTLS_SSL_SERVER_NEW_SESSION_TIC
 
 Handshake, SSL3
 depends_on:MBEDTLS_SSL_PROTO_SSL3:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED
-handshake:"":MBEDTLS_SSL_MINOR_VERSION_0:MBEDTLS_PK_RSA:"":0:0
+handshake:"":MBEDTLS_SSL_MINOR_VERSION_0:MBEDTLS_PK_RSA:"":0:0:MBEDTLS_SSL_MAX_FRAG_LEN_NONE:100:100:1:1
 
+# Note - the case below will have to updated, since the test sends no data due to a 1n-1 split against BEAST, that was not expected when preparing the fragment counting code.
```

Comment:
Should we update this now? If not, when? Do we have an issue filed for this?

Reply:
I propose creating an issue once this PR gets merged, since creating one now would be pointless.

Reply:
We can link to the issue from the code if we make the issue now. It wouldn't be a defect with the current solution, more a TODO. And it's a TODO both before and after this PR merges.

Reply:
Added here: https://github.com/ARMmbed/mbedtls/issues/3070